src/main/java/ml/shifu/shifu/util/CommonUtils.java

```diff
@@ -500,8 +539,8 @@ public boolean apply(ColumnConfig input) {
             reader = ShifuFileUtils.getReader(pathHeader, sourceType);
             pigHeaderStr = reader.readLine();
             if(StringUtils.isEmpty(pigHeaderStr)) {
-                throw new RuntimeException(String.format("Cannot reade header info from the first line of file: %s",
-                        pathHeader));
```

Comment:
Please use shifu-formatter.xml for code format

Reply:
I am already using shifu-formatter.xml to format code. Eclipse gives me this...